examples/visualization/plot_roi_erpimage_by_rt.py

```diff
@@ -0,0 +1,71 @@
+"""
+========================================================================
+Plot single trial activity, EEG channels grouped by ROI and sorted by RT
```

Comment:
title is too long (in HTML viewing) can remove `EEG channels` and retain the same meaning